File: video/rest/rooms/create-group-room/create-group-room.8.x.py
# Download the Python helper library from twilio.com/docs/python/install
import os
from twilio.rest import Client

# Your Account Sid and Auth Token from twilio.com/console
# To set up environmental variables, see http://twil.io/secure
api_key_sid = os.environ['TWILIO_API_KEY']
api_key_secret = os.environ['TWILIO_API_KEY_SECRET']
client = Client(api_key_sid, api_key_secret)

group_room = client.video.rooms.create(
    unique_name='DailyStandup',
    type='group',
    record_participants_on_connect=True,
    status_callback='http://example.org'
)

print(group_room.sid)

File: video/rest/rooms/create-group-room-with-h264/create-group-room.8.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
// To set up environmental variables, see http://twil.io/secure
$apiKeySid = getenv('TWILIO_API_KEY');
$apiKeySecret = getenv('TWILIO_API_KEY_SECRET');
$client = new Client($apiKeySid, $apiKeySecret);

$groupRoom = $client->video->rooms->create([
    'uniqueName' => 'DailyStandupWithH264Codec',
    'type' => 'group',
    'videoCodecs' => 'H264',
    'statusCallback' => 'http://example.org'
]);

echo $groupRoom->sid;
